umbraco/Umbraco-CMS
# Delivery API: Fix default sorting when querying

## Details
Since having all content items left-aligned as the default sort when querying the multi-item endpoint is quite an expensive operation, we decided to keep the index sorting as default. The default sorting will be applied when **no** sort query option is added to the query.

## Test
- Make sure that you can still apply the supported sort options and the result is as intended;
- Verify that if you don't add the 'sort' query option, the items will be returned in the ascending order of when they were last modified.
